docs/gambit/mutation-verifier.md

```diff
@@ -84,20 +108,21 @@ Soon, Certora will enable automatic notifications for asynchronous mutation test
 | Option                         | Description                                                                                                           |
 |:-------------------------------|:----------------------------------------------------------------------------------------------------------------------|
 | `--prover_conf`                | specify the Prover configuration file for verifying mutants                                                           |
-| `--gambit_conf`                | specify the configuration file for mutant generation                                                                  |
+| `--mutation_conf`              | specify the configuration file for mutant generation                                                                  |
 | `--num_mutants`                | request the mutant generator to generate a specific number of mutants. Defaults to 5                                  |
 | `--prover_version`             | specify the version of `certoraRun` to use for verification. Defaults to the installed version of `certoraRun`        |
-| `--server`                     | specify the server environment to run on. Defaults to the value specified in the file of `--prover_conf`, if exists   |
+| `--server`                     | specify the server environment to run on. Defaults to the value specified in the file of `--prover_conf`, if the field exists otherwise whatever `certoraRun` uses by default   |
 | `--debug`                      | show additional logging information during execution                                                                  |
 | `--gambit_out`                 | specify the output directory for gambit. Defaults to a new directory which is added in the working directory          |
-| `--applied_mutants_dir`        | specify the target directory for mutant verification build files. Defaults to a hidden directory used by Prover       |
-| `--ui_out`                     | specify the directory of the mutant verification report JSON used for the web UI                                      |
+| `--applied_mutants_dir`        | specify the target directory for mutant verification build files. Defaults to a hidden Prover internal directory      |
+| `--ui_out`                     | specify a JSON file to dump the mutant verification report used for the web UI                                   |
 | `--dump_link`                  | specify a text file to write the UI report link                                                                       |
+| `--dump_csv`                   | specify a csv file to write the verification JSON report                                                              |
```

Comment:
If it is a csv file, it is not a JSON file - something is contradictory here

Reply:
this was a weird sentence. I meant to say that the content is similar to what is in the JSON. Fixed now.